src/main/java/org/gridsuite/study/server/service/SecurityAnalysisService.java

```diff
@@ -209,10 +209,10 @@ public static SecurityAnalysisParametersValues fromEntity(SecurityAnalysisParame
     public static SecurityAnalysisParametersValues getDefaultSecurityAnalysisParametersValues() {
         SecurityAnalysisParameters securityAnalysisParameters = SecurityAnalysisParameters.load();
         return SecurityAnalysisParametersValues.builder()
-                .lowVoltageAbsoluteThreshold(securityAnalysisParameters.getIncreasedViolationsParameters().getLowVoltageAbsoluteThreshold())
-                .lowVoltageProportionalThreshold(securityAnalysisParameters.getIncreasedViolationsParameters().getLowVoltageProportionalThreshold())
-                .highVoltageAbsoluteThreshold(securityAnalysisParameters.getIncreasedViolationsParameters().getHighVoltageAbsoluteThreshold())
-                .highVoltageProportionalThreshold(securityAnalysisParameters.getIncreasedViolationsParameters().getHighVoltageProportionalThreshold())
+                .lowVoltageAbsoluteThreshold(1)
```

Comment:
You don't want to use constants ? As it is done in Powsybl.

Reply:
Done